external/include/mbedtls/config.h

```diff
@@ -2923,12 +2925,41 @@
 
 #endif							/* MBEDTLS_LIGHT_DEVICE */
 
+#if defined(CONFIG_SE)
+#define MBEDTLS_ENABLE_HARDWARE_ALT
+
+#if defined(CONFIG_HW_RNG)
+#define MBEDTLS_ENTROPY_HARDWARE_ALT
```

Comment:
At line 479, MBEDTLS_ENTROPY_HARDWARE_ALT is commented, but here another configuration for MBEDTLS_ENTROPY_HARDWARE_ALT exists, giving a little poor readability.
So how about adding additional explanation around line 479, otherwise let's get this part closer to that.

Reply:
The commented definition in line 479 should not be used after all. So I moved the description format in these lines, because MBEDTLS_ENTROPY_HARDWARE_ALT depends on CONFIG_SE and CONFIG_HW_RNG determined by the platform and user side by the menuconfig. Thanks for the comment!